extensions/wikia/UserLogin/UserLoginHelper.class.php

```diff
@@ -133,97 +133,82 @@ protected function addUserToUserList( $userId, &$userList ) {
 	}
 
 	/**
-	 * get a list of random popular wikis
-	 * @param integer $require (number of wikis)
-	 * @return array $wikis
+	 * Redirect the user to the appropriate page after login.
+	 *
+	 * @requestParam string returnUrl
 	 */
-	public function getRandomWikis( $require=self::LIMIT_WIKIS ) {
-		return array();
-/**
-		$memKey = wfSharedMemcKey( 'userlogin', 'random_wikis' );
-		$wikis = $this->wg->Memc->get( $memKey );
-		if ( !is_array($wikis) ) {
-			$popularWikis = $this->getPopularWikis();
-			shuffle( $popularWikis );
-			$wikis = array();
-
-			foreach( $popularWikis as $wikiId ) {
-				$themeSettings = WikiFactory::getVarValueByName( 'wgOasisThemeSettings', $wikiId);
-				if( !empty($themeSettings['wordmark-image-url']) ) {
-					$wikis[] = $themeSettings['wordmark-image-url'];
-				}
-
-				if ( count($wikis) >= $require )
-					break;
-			}
-
-			$this->wg->Memc->set( $memKey, $wikis, 60*60*24 );
-		}
-
-		return $wikis;
- **/
+	public function doRedirect() {
```

Comment:
Does this need to also carry optional parameters `$extraReturnToQuery = '', $cbVal = 0 ` to pass on to `getRedirectUrl`?